package.json

```diff
@@ -84,7 +84,7 @@
     "jasmine": "2.3.2",
     "jasmine-core": "2.3.4",
     "jsonfile": "2.2.2",
-    "lodash": "3.5.0",
+    "lodash": "3.10.1",
```

Comment:
While watching travis, could see babel wants updated version still try to work with old version of 3.5 with warning. Bumped up version to babel satisfied.